src/meta-srv/src/procedure/region_failover.rs

```diff
@@ -158,6 +158,10 @@ impl RegionFailoverManager {
     }
 
     pub(crate) async fn do_region_failover(&self, failed_region: &RegionIdent) -> Result<()> {
+        if self.is_maintenance_node().await? {
```

Comment:
Should it be `mode`?
```suggestion
        if self.is_maintenance_mode().await? {
```